components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/CertificateValidationUtil.java

```diff
@@ -424,29 +469,24 @@ private static void addDefaultCACertificates(OMElement trustStoresElement, OMEle
 
         try {
             Iterator trustStoreIterator = trustStoresElement.getChildElements();
-            Registry registry = getGovernanceRegistry(tenantDomain);
             List<X509Certificate> trustedCertificates = new ArrayList<>();
 
             while (trustStoreIterator.hasNext()) {
                 getAllTrustedCerts(trustStoreIterator, trustedCertificates);
             }
 
-            for (X509Certificate certificate : trustedCertificates) {
-                String caCertRegPath = getCACertRegFullPath(certificate);
-                if (log.isDebugEnabled()) {
-                    log.debug("CA certificate registry path: " + caCertRegPath);
-                }
-                addDefaultCACertificateInRegistry(registry, caCertRegPath, certificate, validatorChildElement);
-            }
+            loadCaCertsFromRegistry(trustedCertificates.get(0));
```

Comment:
Why this is needed in `addDefaultCACertificates()` flow?

Reply:
mistake, used it to debug a flow.